docs/src/index.md

```diff
@@ -193,7 +193,292 @@ Since we haven't created our own project yet, we are in the default project, loc
 To return to the `julia>` prompt, either press backspace when the input line is empty or press Ctrl+C.
 Help is available by calling `pkg> help`.
 
-To generate files for a new project, use `pkg> generate`.
+
+### Adding packages
+
+There are two ways of adding packages, either using the `add` command or the `dev` command.
+The most frequently used one is `add` and its usage is described first.
+
+#### Adding registered packages
+
+```
```

Comment:
Prepend something like "In the Pkg REPL packages can be added with the `add` command followed by the name of the package, for example:"